### DISCUSSION TITLE
Ubuntu22.04 无法使用视频下载功能

### DISCUSSION THREAD

**Discussion by leizhenyu-lzy** [the1812/Bilibili-Evolved#4005] [提问 / Q&A] [answered]
想要请教一下
Ubuntu22.04，在edge浏览器，似乎无法使用视频下载功能。是暂时不支持，还是要做什么设置呢？
![image](https://user-images.githubusercontent.com/54441823/219851024-3b180e1f-294b-4225-a0b0-0a8a95d5038a.png)

**Comment by the1812** [comment, marked as answer]
换 StreamSaver 输出

**Comment by leizhenyu-lzy** [comment]
感谢

Grant Howard ***@***.***> 于2023年2月19日周日 20:34写道：
